File: Store.BusinessLogic/Models/Authors/AuthorModelItem.cs
using System.Collections.Generic;
using Store.BusinessLogic.Models.Base;
using Store.BusinessLogic.Models.PrintingEditions;

namespace Store.BusinessLogic.Models.Authors
{
    public class AuthorModelItem : BaseModel
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public IList<PrintingEditionModelItem> PrintingEditions { get; set; } = new List<PrintingEditionModelItem>();
    }
}

File: Store.View/src/app/shared/models/author/author.model.item.ts
import { BaseModel } from 'src/app/shared/models/base/base.model';
import { PrintingEditionModelItem } from 'src/app/shared/models/printing-edition/printing-edition.model.item';

export class AuthorModelItem extends BaseModel {
    public id?: number;
    public name?: string;
    public printingEditions?: Array<PrintingEditionModelItem>;
}
